compile image `hybridsql:0.4.x` won't compile on latest code

**Bug Description**
-- CMAKE_BUILD_TYPE: RelWithDebInfo
-- Found LLVM 9.0.1
-- Using LLVMConfig.cmake in: /deps/usr/lib/cmake/llvm
-- Using LLVM components: LLVMSupport;LLVMCore;LLVMOrcJIT;LLVMX86CodeGen;LLVMX86Desc;LLVMX86Info
-- Found Boost: /deps/usr/include (found version "1.69.0") found components: filesystem date_time regex
-- Found Protobuf Libraries: /deps/usr/lib/libprotobuf.a;-lpthread
-- Boost libraries /deps/usr/lib/libboost_filesystem.a;/deps/usr/lib/libboost_date_time.a;/deps/usr/lib/libboost_regex.a
-- OpenSSL libraries /deps/usr/lib/libssl.a;/deps/usr/lib/libcrypto.a;-lpthread;dl
-- Found SWIG: /deps/usr/bin/swig (found version "4.0.1")
-- Found GTest: /deps/usr/lib/cmake/GTest/GTestConfig.cmake (found version "1.11.0")
-- Found the following ICU libraries:
--   i18n (required)
--   io (required)
--   uc (required)
--   data (required)
-- Failed to find all ICU components (missing: ICU_INCLUDE_DIR)
-- ICU package Not Found
-- Found ICU Libraries: /deps/usr/lib/libicui18n.a;/deps/usr/lib/libicuio.a;/deps/usr/lib/libicuuc.a;/deps/usr/lib/libicudata.a
-- Add src and testcases from base
-- Add src and testcases from udf
-- Add src and testcases from node
-- Add src and testcases from plan
-- Add src and testcases from planv2
-- Add src and testcases from codegen
-- Add src and testcases from vm
-- Add src and testcases from codec
-- Add src and testcases from case
-- Add src and testcases from passes
-- Add src and testcases from storage
-- Add src and testcases from dbms
-- Add src and testcases from tablet
-- Add src and testcases from testing
-- Found LLVM 9.0.1
-- Using LLVMConfig.cmake in: /deps/usr/lib/cmake/llvm
-- Using LLVM components: LLVMSupport;LLVMCore;LLVMOrcJIT;LLVMX86CodeGen;LLVMX86Desc;LLVMX86Info
-- Found the following ICU libraries:
--   i18n (required)
--   io (required)
--   uc (required)
--   data (required)
-- Failed to find all ICU components (missing: ICU_INCLUDE_DIR)
-- ICU package Not Found
-- Found ICU Libraries: /deps/usr/lib/libicui18n.a;/deps/usr/lib/libicuio.a;/deps/usr/lib/libicuuc.a;/deps/usr/lib/libicudata.a
-- CMAKE_INSTALL_PREFIX: /root/OpenMLDB/openmldb
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
RocksDB_LIBRARY
    linked by target "openmldb" in directory /root/OpenMLDB/src

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
make: *** [Makefile:119: configure] Error 1
[root@169e89181b7d OpenMLDB]# uname -a
Linux 169e89181b7d 5.10.104-linuxkit #1 SMP Wed Mar 9 19:05:23 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
[root@169e89181b7d OpenMLDB]# make NPROC=4
[deps]: thirdparty up-to-date. reinstall zetasql from 0.2.6 to 0.2.10
make[1]: Entering directory '/root/OpenMLDB'
cmake -S third-party -B /root/OpenMLDB//.deps -DSRC_INSTALL_DIR=/deps/src -DDEPS_INSTALL_DIR=/deps/usr
-- Install bundled dependencies into /deps/usr
-- Download pre-compiled zetasql from https://github.com/4paradigm/zetasql/releases/download/v0.2.10/libzetasql-0.2.10-linux-gnu-x86_64-centos.tar.gz
-- Download pre-compiled hybridsql assert from https://github.com/4paradigm/hybridsql-asserts/releases/download/v0.5.0.b1/thirdparty-0.5.0.b1-linux-gnu-x86_64-centos.tar.gz
-- Configuring done
-- Generating done
-- Build files have been written to: /root/OpenMLDB/.deps
make[1]: Leaving directory '/root/OpenMLDB'
gmake[1]: Entering directory '/root/OpenMLDB/.deps'
gmake[2]: Entering directory '/root/OpenMLDB/.deps'
gmake[3]: Entering directory '/root/OpenMLDB/.deps'
gmake[4]: Entering directory '/root/OpenMLDB/.deps'
gmake[4]: Leaving directory '/root/OpenMLDB/.deps'
[100%] Built target zetasql
gmake[3]: Leaving directory '/root/OpenMLDB/.deps'
gmake[2]: Leaving directory '/root/OpenMLDB/.deps'
gmake[1]: Leaving directory '/root/OpenMLDB/.deps'
cmake -S . -B /root/OpenMLDB//build -DCMAKE_PREFIX_PATH=/deps/usr -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_BUILD_TYPE=RelWithDebInfo
-- CMAKE_BUILD_TYPE: RelWithDebInfo
-- MAC_TABLET_ENABLE: ON
-- Found Protobuf Libraries: /deps/usr/lib/libprotobuf.a;-lpthread
CMake Warning at CMakeLists.txt:108 (find_package):
  By not providing "FindRocksDB.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "RocksDB", but
  CMake did not find one.

  Could not find a package configuration file provided by "RocksDB" with any
  of the following names:

    RocksDBConfig.cmake
    rocksdb-config.cmake

  Add the installation prefix of "RocksDB" to CMAKE_PREFIX_PATH or set
  "RocksDB_DIR" to a directory containing one of the above files.  If
  "RocksDB" provides a separate development package or SDK, be sure it has
  been installed.


-- CMAKE_BUILD_TYPE: RelWithDebInfo
-- Found LLVM 9.0.1
-- Using LLVMConfig.cmake in: /deps/usr/lib/cmake/llvm
-- Using LLVM components: LLVMSupport;LLVMCore;LLVMOrcJIT;LLVMX86CodeGen;LLVMX86Desc;LLVMX86Info
-- Found Protobuf Libraries: /deps/usr/lib/libprotobuf.a;-lpthread
-- Boost libraries /deps/usr/lib/libboost_filesystem.a;/deps/usr/lib/libboost_date_time.a;/deps/usr/lib/libboost_regex.a
-- OpenSSL libraries /deps/usr/lib/libssl.a;/deps/usr/lib/libcrypto.a;-lpthread;dl
-- Found the following ICU libraries:
--   i18n (required)
--   io (required)
--   uc (required)
--   data (required)
-- Failed to find all ICU components (missing: ICU_INCLUDE_DIR)
-- ICU package Not Found
-- Found ICU Libraries: /deps/usr/lib/libicui18n.a;/deps/usr/lib/libicuio.a;/deps/usr/lib/libicuuc.a;/deps/usr/lib/libicudata.a
-- Add src and testcases from base
-- Add src and testcases from udf
-- Add src and testcases from node
-- Add src and testcases from plan
-- Add src and testcases from planv2
-- Add src and testcases from codegen
-- Add src and testcases from vm
-- Add src and testcases from codec
-- Add src and testcases from case
-- Add src and testcases from passes
-- Add src and testcases from storage
-- Add src and testcases from dbms
-- Add src and testcases from tablet
-- Add src and testcases from testing
-- Found LLVM 9.0.1
-- Using LLVMConfig.cmake in: /deps/usr/lib/cmake/llvm
-- Using LLVM components: LLVMSupport;LLVMCore;LLVMOrcJIT;LLVMX86CodeGen;LLVMX86Desc;LLVMX86Info
-- Found the following ICU libraries:
--   i18n (required)
--   io (required)
--   uc (required)
--   data (required)
-- Failed to find all ICU components (missing: ICU_INCLUDE_DIR)
-- ICU package Not Found
-- Found ICU Libraries: /deps/usr/lib/libicui18n.a;/deps/usr/lib/libicuio.a;/deps/usr/lib/libicuuc.a;/deps/usr/lib/libicudata.a
-- CMAKE_INSTALL_PREFIX: /root/OpenMLDB/openmldb
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
RocksDB_LIBRARY
    linked by target "openmldb" in directory /root/OpenMLDB/src

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
make: *** [Makefile:119: configure] Error 1


**Expected Behavior**



**Steps to Reproduce**

1. docker pull 4pdosc/hybridsql:0.4.4
2. docker run -it 4pdosc/hybridsql:0.4.4 bash
3. cd ~
4. git clone https://github.com/4paradigm/OpenMLDB.git
5. cd OpenMLDB
6.make  【
-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
make: *** [Makefile:119: configure] Error 1
】

--- a/docs/en/deploy/compile.md
+++ b/docs/en/deploy/compile.md
@@ -8,23 +8,25 @@ Build
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
+:bulb: Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridql:latest` image instead.
+
 1. Pull the docker image
 
    ```bash
-    docker pull 4pdosc/hybridsql:0.4.3
+    docker pull 4pdosc/hybridsql:0.4.4
    ```
 
 2. Create a docker container with the hybridsql docker image
 
    ```bash
-   docker run -it 4pdosc/hybridsql:0.4.3 bash
+   docker run -it 4pdosc/hybridsql:0.4.4 bash
    ```
 
-3. Download the OpenMLDB source code inside docker container
+3. Download the OpenMLDB source code inside docker container, and setting the branch into v0.4.4
 
    ```bash
    cd ~
-   git clone https://github.com/4paradigm/OpenMLDB.git
+   git clone -b v0.4.4 https://github.com/4paradigm/OpenMLDB.git
    ```
 
 4. Compile OpenMLDB

--- a/docs/en/deploy/compile.md
+++ b/docs/en/deploy/compile.md
@@ -8,7 +8,7 @@ Build
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
-Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
+Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrates compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
 
 1. Pull the docker image
 

--- a/docs/en/deploy/compile.md
+++ b/docs/en/deploy/compile.md
@@ -8,7 +8,7 @@ Build
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
-:bulb: Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridql:latest` image instead.
+:bulb: Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
 
 1. Pull the docker image
 

--- a/docs/en/deploy/compile.md
+++ b/docs/en/deploy/compile.md
@@ -8,7 +8,7 @@ Build
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
-:bulb: Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
+Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrate compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
 
 1. Pull the docker image
 

--- a/docs/en/deploy/compile.md
+++ b/docs/en/deploy/compile.md
@@ -8,7 +8,7 @@ Build
 This section describes the steps to compile and use OpenMLDB inside its official docker image [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql).
 The docker image has packed required tools and dependencies, so there is no need to setup them separately. To compile without the official docker image, refer to the section [Detailed Instructions for Build](#detailed-instructions-for-build) bellow.
 
-Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This setion demonstrates compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
+Keep in mind that you should always use the same version of both compile image and [OpenMLDB version](https://github.com/4paradigm/OpenMLDB/releases). This section demonstrates compiling for [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) under `hybridsql:0.4.4` ，If you prefer to compile on the latest code in `main` branch, pull `hybridsql:latest` image instead.
 
 1. Pull the docker image
 

--- a/docs/zh/deploy/compile.md
+++ b/docs/zh/deploy/compile.md
@@ -4,7 +4,7 @@
 
 此节介绍在官方编译镜像 [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql) 中编译 OpenMLDB。镜像内置了编译所需要的工具和依赖，因此不需要额外的步骤单独配置它们。关于基于非 docker 的编译使用方式，请参照下面的 [编译详细说明](#编译详细说明) 章节。
 
-:bulb: 关于编译镜像版本，需要注意拉取的镜像版本和 [OpenMLDB 发布版本](https://github.com/4paradigm/OpenMLDB/releases)保持一致。以下例子演示了在 hybridsql 0.4.4 镜像版本上编译 [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) 的代码，如果要编译最新 main 分支的代码，则需要拉取 hybridsql:latest 版本镜像。
+:bulb: 关于编译镜像版本，需要注意拉取的镜像版本和 [OpenMLDB 发布版本](https://github.com/4paradigm/OpenMLDB/releases)保持一致。以下例子演示了在 `hybridsql:0.4.4` 镜像版本上编译 [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) 的代码，如果要编译最新 `main` 分支的代码，则需要拉取 `hybridsql:latest` 版本镜像。
 
 1. 下载 docker 镜像
     ```bash

--- a/docs/zh/deploy/compile.md
+++ b/docs/zh/deploy/compile.md
@@ -4,7 +4,7 @@
 
 此节介绍在官方编译镜像 [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql) 中编译 OpenMLDB。镜像内置了编译所需要的工具和依赖，因此不需要额外的步骤单独配置它们。关于基于非 docker 的编译使用方式，请参照下面的 [编译详细说明](#编译详细说明) 章节。
 
-:bulb: 关于编译镜像版本，需要注意拉取的镜像版本和 [OpenMLDB 发布版本](https://github.com/4paradigm/OpenMLDB/releases)保持一致。以下例子演示了在 `hybridsql:0.4.4` 镜像版本上编译 [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) 的代码，如果要编译最新 `main` 分支的代码，则需要拉取 `hybridsql:latest` 版本镜像。
+关于编译镜像版本，需要注意拉取的镜像版本和 [OpenMLDB 发布版本](https://github.com/4paradigm/OpenMLDB/releases)保持一致。以下例子演示了在 `hybridsql:0.4.4` 镜像版本上编译 [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) 的代码，如果要编译最新 `main` 分支的代码，则需要拉取 `hybridsql:latest` 版本镜像。
 
 1. 下载 docker 镜像
     ```bash

--- a/docs/zh/deploy/compile.md
+++ b/docs/zh/deploy/compile.md
@@ -4,6 +4,8 @@
 
 此节介绍在官方编译镜像 [hybridsql](https://hub.docker.com/r/4pdosc/hybridsql) 中编译 OpenMLDB。镜像内置了编译所需要的工具和依赖，因此不需要额外的步骤单独配置它们。关于基于非 docker 的编译使用方式，请参照下面的 [编译详细说明](#编译详细说明) 章节。
 
+:bulb: 关于编译镜像版本，需要注意拉取的镜像版本和 [OpenMLDB 发布版本](https://github.com/4paradigm/OpenMLDB/releases)保持一致。以下例子演示了在 hybridsql 0.4.4 镜像版本上编译 [OpenMLDB v0.4.4](https://github.com/4paradigm/OpenMLDB/releases/tag/v0.4.4) 的代码，如果要编译最新 main 分支的代码，则需要拉取 hybridsql:latest 版本镜像。
+
 1. 下载 docker 镜像
     ```bash
     docker pull 4pdosc/hybridsql:0.4.4
@@ -14,10 +16,10 @@
     docker run -it 4pdosc/hybridsql:0.4.4 bash
     ```
 
-3. 在 docker 容器内, 克隆 OpenMLDB
+3. 在 docker 容器内, 克隆 OpenMLDB, 并切换分支到 v0.4.4
     ```bash
     cd ~
-    git clone https://github.com/4paradigm/OpenMLDB.git
+    git clone -b v0.4.4 https://github.com/4paradigm/OpenMLDB.git
     ```
 
 4. 在 docker 容器内编译 OpenMLDB

